qa-ibd.planx-pla.net/portal/gitops.json

```diff
@@ -243,6 +243,7 @@
       ],
       "guppyConfig": {
         "dataType": "participant",
+        "tierAccessLevel": "regular",
```

Comment:
i don't think we need it in this file?

Reply:
https://github.com/uc-cdis/data-portal/blob/master/docs/portal_config.md?plain=1#L336

Reply:
Also Mingfei confirmed it 
https://cdis.slack.com/archives/CNAHGS3MZ/p1655131491082749?thread_ts=1655131112.282539&cid=CNAHGS3MZ